python/nav/alertengine/dispatchers/sms_dispatcher.py

```diff
@@ -71,9 +74,18 @@ def get_fallback_message(self, alert, language, message_type):
 
         return '%s: No sms message for %d' % (alert.netbox, alert.id)
 
+    class PhoneNumberForm(forms.Form):
+        phone_number = PhoneNumberField()
+
     @staticmethod
     def is_valid_address(address):
-        if address.isdigit():
-            return True
-        else:
+        if address.startswith("00"):
+            # needs to be replaced since phone number library requires the plus
+            address = "+" + address[2:]
+        elif not address.startswith("+"):
+            address = "+47" + address
```

Comment:
I don't approve of this.  This will break for all NAV users outside of Norway, which would now be forced to update their alert profile addresses to keep things working.

National phone numbers without an international prefix should still work.

Reply:
The library we use in Argus requires country codes, so without country codes we can't really guess where the number is from and can't rely on libraries for validation. How should we do the validation instead?

Reply:
Argus has an advantage NAV doesn't have: It has required phone numbers to have an international prefix more or less from its first release, so there are no users with anything else in their profile.

_I'd say the simplest approach is the naive one: Allow a phone number to be any string of digits, optionally preceded by a `+` sign._

There may potentially be other values used in the wild that work just fine with SMSD/Gammu, that this phone number validation library, our any new code we introduce, might reject.  What about phone number values with spaces or dashes in them? These could work with Gammu, but could also be rejected in the future if we add validation code.


Any other approach than to allow what's in the wild already would require us to provide the user with some sort of migration path so they can get all their user's phone numbers formatted correctly before they spin up a new version of NAV, and that's also hassle I would like to avoid if we can :)